packages/go_router/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 - Updates minimum supported SDK version to Flutter 3.27/Dart 3.6.
 - Fixes typo in API docs.
 
+## 15.1.3
```

Comment:
The changes above this have already been published, we cannot insert another version retroactively.